app/src/features/projects/components/ProjectGeneralInformationForm.tsx

```diff
@@ -179,11 +188,15 @@ const ProjectGeneralInformationForm: React.FC = () => {
 
   const config = useContext(ConfigContext);
 
-  // const { values, setFieldValue, setFieldError } = formikProps;
-  // console.log('values', values);
+  const { setFieldValue } = formikProps;
 
   const [image, setImage] = useState('' as any);
 
+  // When the image is updated make sure to update the formik field
+  useEffect(() => {
```

Comment:
Seems unnecessary to use a useEffect to update the formik. This could be acheived by assigning the formik value when the image is uploaded.
Reducing potential rerenders on the page is important and could cause other issues later in development

Reply:
I actually had it that way originally. It was causing some weird issues.

Reply:
It works well the way it is. Sorry.

Reply:
I'll make a ticket.